### PR TITLE
added ons-toolbar karma test

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: core/elements/ons-toolbar.spec.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      expect(element.style.left).not.to.equal('0px');
+      expect(element.style.right).not.to.equal('0px');
+      expect(element.style.top).not.to.equal('0px');
+    }
+  });
+  */
+
+  it('recognizes Android platform', function() {
+    var element = new OnsToolbarElement();
+    if(ons.platform.isAndroid() && !this.hasAttribute('fixed-style'))
```
Comment (argelius) [inline core/elements/ons-toolbar.spec.js]: Always use brackets for statements

Comment (argelius) [inline core/elements/ons-toolbar.spec.js]: `if` should be followed by space

**Comment by andipavllo** [comment]
@argelius fixed

**Comment by argelius** [comment]
Thanks @andipavllo !

**Pull request merged**